iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ThreeWindingsTransformer.java

```diff
@@ -210,7 +210,7 @@
      *     </tbody>
      * </table>
      */
-    public interface Leg extends RatioTapChangerHolder, PhaseTapChangerHolder {
+    public interface Leg extends RatioTapChangerHolder, PhaseTapChangerHolder, FlowsLimitsHolder {
```

Comment:
I am not sure to get what you mean here with that `FlowLimitsHolder`: for you, only current limits can be associated to a three winding transformers ?

Reply:
It is just the default behaviour?